CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # Changelog
 
+## Unreleased
+
+### Dependencies
+
+- Bump Sentry Android SDK to `6.4.3` ([#230](https://github.com/getsentry/sentry-capacitor/pull/230))
```

Comment:
I think it would be better to keep referring to the dependency as sentry-java, so as not to confuse ppl who are not aware that this is the Android SDK